### ISSUE TITLE
error in loss.backward() project

### ISSUE THREAD

**Issue by kobrafarshidi** [uakarsh/latr#5]
Hi again .In the continuation of run of the pre-train section in the 14th block I I have encountered an error. I would appreciate if u help
![e3](https://user-images.githubusercontent.com/91652559/191094898-89503fb4-01ea-44c5-86bc-3d8cac9b085e.PNG)
 me with this too.

**Comment by uakarsh** [comment]
Actually, I have seen this error for the first time.  Did you run the block more than 1 time?

**Comment by kobrafarshidi** [comment]
actually, for the first time I had this error but after a few times this error disappeared thank you so much again for your guidance.

**Comment by uakarsh** [comment]
No worries. Hope you find the work useful

**Comment by kobrafarshidi** [comment]
Thank you I really appreciate your help
